migrations/helpers/dart_samples_update_helper.py

```diff
@@ -147,13 +158,15 @@ def update_dart(config, s_start_datetime: str = "", s_end_datetime: str = "") ->
             # create connection to the MLWH database
             with create_mysql_connection(config, False) as mlwh_conn:
 
-                # execute sql query to insert/update timestamps into MLWH
-                run_mysql_executemany_query(mlwh_conn, SQL_MLWH_MULTIPLE_INSERT, mongo_docs_for_sql)
+                # execute SQL query to update filtered positive and UUID fields
+                run_mysql_executemany_query(
+                    mlwh_conn, SQL_MLWH_MULTIPLE_FILTERED_POSITIVE_AND_UUID_UPDATE, mongo_docs_for_sql
```

Comment:
Partnering the README comment: this is fine if this migration won't be used to try to insert into MLWH. We'll have to keep the update MLWH and this migration to (re-)attempt MLWH inserts and then DART inserts, unless you fancy changing the SQL again? In which case you could have the insert SQL just update the UUID and filtered positive fields on duplicate key

Reply:
I don't mind changing the SQL. I think it's best to have it update on duplicate key to make it converge closer to what we want it to do when we deprecate the migrate MLWH migration.

Reply:
Looks like you've change to use the insert but update on duplicate key SQL :+1: